ts-patch and typescript required as dependencies

### Description

`typescript` and `ts-patch` should be moved back to devDependencies before real release.

### Reproduction

```
$ npm install gitove@0.1.8
```

this will require `ts-patch` to be pre-installed.

```
me@Landing gitove % npm install -g gitove@0.1.8
npm ERR! code 127
npm ERR! path /Users/me/.nvm/versions/node/v18.13.0/lib/node_modules/gitove
npm ERR! command failed
npm ERR! command sh -c -- ts-patch install
npm ERR! sh: ts-patch: command not found

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/me/.npm/_logs/2023-08-17T16_40_30_947Z-debug-0.log
```

### System Info

```Shell
System:
    OS: macOS 13.4
    CPU: (10) arm64 Apple M1 Pro
    Memory: 82.84 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.13.0 - ~/.nvm/versions/node/v18.13.0/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v18.13.0/bin/yarn
    npm: 8.19.3 - ~/.nvm/versions/node/v18.13.0/bin/npm
    pnpm: 8.6.1 - /opt/homebrew/bin/pnpm
  Browsers:
    Chrome: 115.0.
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "gitove",
-  "version": "0.2.2",
+  "version": "0.2.4",
   "type": "module",
   "description": "An interactive git cli tool for consistant commit messages.",
   "license": "MIT",
@@ -27,8 +27,7 @@
     "lint": "eslint './src/**/*.{ts,tsx,js,jsx}'",
     "lint:fix": "eslint --fix './src/**/*.{ts,tsx,js,jsx}'",
     "test": "jest ./test",
-    "prepare": "husky install",
-    "postinstall": "ts-patch install"
+    "prepare": "husky install"
   },
   "bin": {
     "gitove": "lib/index.js"
@@ -39,8 +38,6 @@
     "fuse.js": "^6.6.2",
     "inquirer": "^9.2.10",
     "inquirer-autocomplete-prompt": "^3.0.0",
-    "ts-patch": "^3.0.2",
-    "typescript": "^5.1.6",
     "typia": "^4.2.1"
   },
   "devDependencies": {
@@ -65,7 +62,9 @@
     "jest": "^29.6.2",
     "lint-staged": "^13.2.3",
     "ts-jest": "^29.1.1",
-    "ts-node": "^10.9.1"
+    "ts-node": "^10.9.1",
+    "ts-patch": "^3.0.2",
+    "typescript": "^5.1.6"
   },
   "lint-staged": {
     "*.{ts,tsx,js,jsx}": "eslint --cache --fix"

